src/server_manager/web_app/ui_components/outline-intro-step.js

```diff
@@ -276,20 +278,24 @@ Polymer({
     },
     isDigitalOceanAccountConnected: {
       type: Boolean,
-      computed: '_computeIsDigitalOceanAccountConnected(digitalOceanAccountName)',
+      computed: '_computeIsAccountConnected(digitalOceanAccountName)',
     },
     gcpAccountName: {
       type: String,
       value: null,
     },
+    isGcpAccountConnected: {
```

Comment:
Same, call the function in the template instead

Reply:
Done